app/src/main/java/com/github/palFinderTeam/palfinder/ui/login/LoginActivity.kt

```diff
@@ -46,7 +62,7 @@ class LoginActivity : AppCompatActivity() {
     private fun reload() {
```

Comment:
Is it normal that the function is empty ?

Reply:
It is an artefact I forgot to remove, sorry